oval_graph/arf_xml_parser/arf_xml_parser.py

```diff
@@ -12,14 +12,16 @@
 from ._oval_scan_definitions import _OVALScanDefinitions
 from .global_namespaces import namespaces
 
+LOCAL_DATA_DIR = Path(__file__).parent.joinpath('../schemas/')
```

Comment:
Don't use `joinpath` - the `/` operator does the same and is more elegant, i.e. `Path(__file__).parent.parent / "schemas"`